main.go

```diff
@@ -37,6 +38,7 @@ func main() {
 	flag.BoolVar(&flags.skipEnsure, "skip-ensure", false,
 		"suppress mock implementation check, avoid import cycle if mocks generated outside of the tested package")
 	flag.BoolVar(&flags.remove, "rm", false, "first remove output file, if it exists")
+	flag.BoolVar(&flags.force, "force", false, "force generation, otherwise check if go generate file is newer than output file")
```

Comment:
This change is not backwards compatible. Maybe it is better to preserve the current mode of operation and add a flag to opt-in for the `mtime` behavior. Maybe the `-mtime` flag could be used to enable the `mtime` mode as well as contain the name of the source file as well. If the source file is the empty string, it would fall back to `GOFILE` env variable.